grid.css

```diff
@@ -0,0 +1,26 @@
+
+.row {
+    display: flex; 
+    padding: 80px 0; 
+}
+
+[class *= 'col-'] {
+    padding: 0 20px; 
+}
+.col-3 {
+    width: 25%; 
+}
+
+.col-4 {
+    width: calc(100% / 3); 
+}
+.col-6 {
+    width: 50%
+}
+.col-8 {
+    width: calc(100% / 12 *8); 
+}
+.col-9 {
+    width: calc( 100% / 3 * 3); 
+}
+
```

Comment:
Lege dir im Grid wirklich alle ".col-" vollständig an - damit du die Klassen in der index nur noch vergeben musst